src/diffusers/models/transformers/transformer_mochi.py

```diff
@@ -202,7 +478,10 @@ def _get_positions(
         return positions
 
     def _create_rope(self, freqs: torch.Tensor, pos: torch.Tensor) -> torch.Tensor:
-        freqs = torch.einsum("nd,dhf->nhf", pos, freqs.float())
+        with torch.autocast(freqs.device.type, enabled=False):
```

Comment:
what does this do here? are we disable autocast here in case it is enabled 
ok here if this is what causing the quality issue - but we should come up with something better :)

Reply:
Hmm actually I think with all the manual casts we have in place, just setting `torch_dtype=torch.bfloat16` should allow us to effectively run Mochi as if autocast is enabled. 

We could just remove this autocast context manager. Will test.